src/coreclr/nativeaot/Runtime/HandleTableHelpers.cpp

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 //
-// Helper functions that are p/invoked from redhawkm in order to expose handle table functionality to managed
+// Helper functions that are p/invoked from NativeAOTm in order to expose handle table functionality to managed
```

Comment:
`NativeAOTm` is not a thing. This is example, of place that needs manual tweaks.